### PR TITLE
JAX-WS: Share ComponentMetaData with child threads

### PR DESCRIPTION
**Pull request by neuwerk**
This PR does the following:

- Allows ComponentMetaData to be shared with child threads when running Async/`@OneWay` incoming JAX-WS requests.

- Adds a new fat bucket `io.openliberty.jaxws.beanvalidation_fat`  with three new test suites:

1. `BeanValidationTest` which tests Bean Validation on a JAX-WS Endpoint using the `Configuration` API to obtain a `ValidatorFactory` instance. 
2. `InjectionValidationTest` which tests Bean Validation on a JAX-WS Endpoint using `@Resource` injected `Validator/ValidatorFactory` instances.
3. `CDIValidationTest` which tests Bean Validation on a JAX-WS Endpoint using `@Inject` injected `Validator/ValidatorFactory` instances.

### CODE REVIEW

**Comment by neuwerk** [comment]
#build

File: dev/com.ibm.ws.jaxws.2.3.common/src/com/ibm/ws/jaxws/threading/LibertyJaxWsAutomaticWorkQueueImpl.java [hunk 1]
```diff
@@ -13,33 +13,47 @@
 package com.ibm.ws.jaxws.threading;
```
Comment (BerksanAtes) [inline dev/com.ibm.ws.jaxws.2.3.common/src/com/ibm/ws/jaxws/threading/LibertyJaxWsAutomaticWorkQueueImpl.java:13]: Copyright 2019, 2024

**Review by BerksanAtes — APPROVED** [review]
As long as PB comes clean, it looks good to me!

**Comment by neuwerk** [comment]
#libby

**Pull request merged**